lib/containers/utils.pm

```diff
@@ -315,4 +315,78 @@ sub reset_container_network_if_needed {
     }
 }
 
+sub test_volumes {
+    my %args = @_;
+    my $runtime = $args{runtime};
+
+    # Create a directory with the sticky-bit set
+    assert_script_run('mkdir -m 1777 test');
+
+    # Create Dockerfile with VOLUME defined
+    assert_script_run(q{echo -e 'FROM busybox\nVOLUME /test' > test/Dockerfile});
+
+    # Build image and load it
+    assert_script_run("$runtime build -t test -f test/Dockerfile test/");
+
+    assert_script_run('touch test/test');
+
+    # Test --volumes-from
+    assert_script_run("$runtime run -d --name test test");
+    assert_script_run("$runtime run --rm --volumes-from test test ls /test | grep test");
+    assert_script_run("$runtime rm -vf test");
+
+    assert_script_run("$runtime run -d --name test -v \$PWD/test:/test test");
```

Comment:
```suggestion
    my $mount = '\$PWD/test:/test';
    $mount .= ":Z" if ($runtime eq 'podman');
    assert_script_run("$runtime run -d --name test -v $mount test");
```
@dcermak is right, in the case of podman the mount should include a private SELinux label.

Reply:
I did a cooler hack for that.